tools/bioconductor-msnbase/bioconductor_msnbase_smooth_mz.xml

```diff
@@ -0,0 +1,105 @@
+<tool id="bioconductor_msnbase_smooth_mz" name="bioconductor-msnbase smooth mz" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0" license="MIT">
+    <description>smooth raw MS data in mass-to-charge ratio domain</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <edam_topics>
+        <edam_topic>topic_3170</edam_topic> <!-- Proteomics -->
+        <edam_topic>topic_3391</edam_topic> <!-- Data handling -->
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_3632</edam_operation> <!-- Data smoothing -->
+        <edam_operation>operation_2945</edam_operation> <!-- Data handling -->
+    </edam_operations>
+
+    <expand macro="xrefs"/>
+    <expand macro="creator"/>
+    <expand macro="requirements"/>
+
+    <command detect_errors="exit_code"><![CDATA[
+    Rscript -e 'source("${run_script}")'
```

Comment:
isn't `Rscript '$run_script'` also working?

Reply:
should be